tests/test_main.py

```diff
@@ -8,6 +8,11 @@
 from aiorun import run, shutdown_waits_for, _DO_NOT_CANCEL_COROS
 import pytest
 
+try:
+    from asyncio import all_tasks
```

Comment:
ditto here with tagging this try-except with the python version that introduces `asyncio.all_tasks`